docs/pages/database-access/guides/mysql-self-hosted.mdx

```diff
@@ -18,7 +18,7 @@ on the auth server.
 ```sh
 # Export Teleport's certificate authority and generate certificate/key pair
 # for host db.example.com with a one year validity period.
-$ tctl auth sign --format=db --host=db.example.com --out=server --ttl=8760h
+sudo tctl auth sign --format=db --host=db.example.com --out=server --ttl=1h
```

Comment:
The same for databases. @r0mant What's your thoughts on making this smoother process and managing the Teleport CA for on-prem databases

Reply:
Great, set to `2190h`. I appreciate your input @benarent and @klizhentas!